### DISCUSSION TITLE
Reduce docker image size by combining RUN calls

### DISCUSSION THREAD

**Discussion by pixeline** [usememos/memos#634] [Ideas]
I was wondering if it was an oversight or if there is a reason for not combining these 2 run calls, as it would reduce the docker image size ?
https://github.com/usememos/memos/blob/ab8e3473a191d53f5bb97988ff5d58115fae4c4f/Dockerfile#L7-L8

Thank you!

**Comment by boojack** [comment]
> Reduce docker image size by combining RUN calls

After having a test on my machine, it won't reduce the image size.

**Reply by pixeline** [reply]
Indeed, I double-checked and absolutely no gain. Thanks for checking !
